Library/Formula/gcab.rb

```diff
@@ -1,26 +1,33 @@
-require 'formula'
-
 class Gcab < Formula
-  homepage 'https://wiki.gnome.org/msitools'
-  url 'http://ftp.gnome.org/pub/GNOME/sources/gcab/0.4/gcab-0.4.tar.xz'
-  sha1 'd81dfe35125e611e3a94c0d4def37ebf62b9187c'
+  homepage "https://wiki.gnome.org/msitools"
+  url "http://ftp.acc.umu.se/pub/GNOME/sources/gcab/0.6/gcab-0.6.tar.xz"
+  mirror "https://mirrors.kernel.org/debian/pool/main/g/gcab/gcab_0.6.orig.tar.xz"
+  sha256 "a0443b904bfa7227b5155bfcdf9ea9256b6e26930b8febe1c41f972f6f1334bb"
 
-  depends_on 'intltool' => :build
-  depends_on 'pkg-config' => :build
-  depends_on 'vala' => :build
-  depends_on 'gettext'
-  depends_on 'glib'
-  depends_on 'gobject-introspection'
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vala" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gobject-introspection"
 
   # work around ld not understanding --version-script argument
   # upstream bug: https://bugzilla.gnome.org/show_bug.cgi?id=708257
   patch :DATA
```

Comment:
Doesn't look like this patch is getting accepted upstream. Can you check if it was submitted and, if not, submit it and, if so, see if we can replace it with e.g. an `inreplace` (or see if it's even still needed). Thanks!